plugins/firebase.js

```diff
@@ -0,0 +1,60 @@
+import { initializeApp } from 'firebase/app'
+import { getAuth, createUserWithEmailAndPassword, signInWithEmailAndPassword, onAuthStateChanged, getIdToken, signOut } from 'firebase/auth'
+import { firebaseConfig } from '@/service/options' // Need to be defined. File is untracked now
```

Comment:
It is make hard to provide additional file to container in Deployment pipeline. 
Should use `env` instead